com/course/zenkouv/src/test/resources/testSuit.xml

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+<suite name="Test Suit">
+    <test name="DemoTest" parallel="tests" thread-count="1">
```

Comment:
` parallel="tests"` means that all blocks ` <test> ` will run in parallel
if you want to run your tests in parallel   - you should use parallel="methods"
it is testNg naming magic

Reply:
fix. tnx